src/background.ts

```diff
@@ -120,10 +121,24 @@ const updateInfos = JSON.parse(
   })
 );
 
+// initialize menu
+const menu = MenuBuilder()
+  .setOnLaunchModeItemClicked((useGpu) => {
+    store.set("useGpu", useGpu);
+
+    dialog.showMessageBoxSync(win, {
+      message: "音声合成エンジン起動モードの変更が完了しました",
```

Comment:
ユビキタス言語的には「エンジンの起動モード」になりそうです。
```suggestion
      message: "エンジンの起動モードを変更しました",
```

Reply:
そもそもの日本語がおかしかったですね…サジェストありがとうございます